metsim/methods/mtclim.py

```diff
@@ -59,12 +82,28 @@ def calc_t_air(df: pd.DataFrame, elev: float, params: dict):
 
 
 def calc_prec(df: pd.DataFrame, params: dict):
-    """Adjust precitation according to isoh"""
-    df['prec'] *= (df.get('site_isoh', 1) / df.get('base_isoh', 1))
+    """
+    Adjust precitation according to ratio of
+    isohyet ratio of the given site to some base
+    value.
 
+    Args:
+        df: Dataframe containing daily precipitation
+            timeseries.
+        params: Dictionary containing isohyet values
+    """
+    df['prec'] *= (params['site_isoh'] / params['base_isoh'])
+
+
+def calc_snowpack(df: pd.DataFrame, snowpack=0.0):
```

Comment:
I would like to, they render nicely in the documentation.  Keywords can be done just like anything else: `def fakefun(fakearg: type=default)`.